### DISCUSSION TITLE
Still being translated?

### DISCUSSION THREAD

**Discussion by Nissan-O** [File-New-Project/EarTrumpet#1202] [Q&A] [answered]
Hello, is the project still ment to be translated to other languages or was that aspect abandoned already?

**Comment by riverar** [comment, marked as answer]
@Nissan-O Still accepting translations on CrowdIn (https://crowdin.com/project/eartrumpet). If you don't see a language you want to translate to, let me know and I'll add it.

**Reply by Nissan-O** [reply]
Ok nice, I made progress in the translation to Hebrew. Can I see my Crowdln contributions on github if my accounts are linked?
And what to do when the translation is done?

**Reply by riverar** [reply]
@Nissan-O Saw them and approved them. CrowdIn will now automatically submit a change to this repository and I'll merge them into the `dev` branch shortly. At that point, a build will automatically go out (for users that have EarTrumpet (dev) installed) and it will contain your language changes!

I'm working on getting everyone's names into the README (and possibly the app) for proper credit too!

**Reply by Nissan-O** [reply]
Ok, thank you very much. So does the Microsoft store version of EarTrumpet eventually get the features of EarTrumpet(dev) or the dev is a seperate version?
Sorry for all of this, I'm new on github.

**Reply by riverar** [reply]
@Nissan-O The Store version eventually gets it when I'm happy with `dev` and merge it into `master`. The dev version is separate and can be found at https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller (on the front of our https://eartrumpet.app webpage).

**Reply by Nissan-O** [reply]
Hello, the Hebrew translation for EarTrumpet is done! please verify my changes on crowdin. Also, in the MS store preview I made a change to a previously verefied translation, I think that the name "EarTrumpet" should stay as is even in Hebrew, the translation to Hebrew sounds really bad. Thanks for making the program

**Reply by riverar** [reply]
@Nissan-O Thanks! Approved and it's now in the dev branch. It should show up in the dev version (as an update) of the software in a few minutes (https://github.com/File-New-Project/EarTrumpet/runs/7809064412?check_suite_focus=true). To update, restart EarTrumpet twice.
